Extension/src/LanguageServer/settings.ts

```diff
@@ -253,7 +254,9 @@ export class CppSettings extends Settings {
     public get defaultCustomConfigurationVariables(): { [key: string]: string } | undefined { return super.Section.get<{ [key: string]: string }>("default.customConfigurationVariables"); }
     public get useBacktickCommandSubstitution(): boolean | undefined { return super.Section.get<boolean>("debugger.useBacktickCommandSubstitution"); }
     public get codeFolding(): boolean { return super.Section.get<string>("codeFolding") === "Enabled"; }
-    public get legacyCompilerArgsBehavior(): boolean | undefined  { return super.Section.get<boolean>("legacyCompilerArgsBehavior"); }
+    public get caseSensitiveFileSupport(): boolean { return super.Section.get<string>("caseSensitiveFileSupport") === "enabled"; }
```

Comment:
If we're doing this as a boolean, then I think we should force it to true for linux/mac.  Basicially, `!util.isWindows() || setting === 'enabled'`

Reply:
I didn't see any code on the native side to deal with this.

Reply:
Sounds good